frameworks/PHP/webman/benchmark_config.json

```diff
@@ -20,7 +39,7 @@
       "webserver": "None",
       "os": "Linux",
       "database_os": "Linux",
-      "display_name": "webman",
+      "display_name": "webman-pgsql",
```

Comment:
We are also changing the display_name, for easier visibility.

From "webman-pgsql" to "webman [pgsql]"

Or if we have more options like in Workerman :  "workerman [JIT, pgsql]"